site-scrapers/Color/index.js

```diff
@@ -46,43 +46,56 @@ async function ScrapeWebsiteData(siteName, siteUrl) {
 
 function formatResponse(availabilityResponse) {
     const availability = JSON.parse(availabilityResponse).results;
-    const results = { availability: {}, hasAvailability: false };
-    /* COMMENTING OUT BECAUSE IT'S NOW INVITE-ONLY */
-    // Collect availability count by date
-    //  availability.reduce((memo, currentValue) => {
-    //     /* The availability returns an array of appointments like this:
-    //         {
-    //             "start": "2021-02-22T14:00:00+00:00",
-    //             "end": "2021-02-22T14:04:00+00:00",
-    //             "capacity": 1,
-    //             "remaining_spaces": -1
+    const results = {
+        massVax: true, // This is a MassVax site that only allows preregistration
```

Comment:
We might want to change this field to `inviteOnly` or `inviteNeeded`, so that if other sites (perhaps not massVax ones) use an invite-only system, we can add that field without causing confusion. Or perhaps some massVax sites won't be invite-only in the future. (No idea how likely either of these scenarios are).

Reply:
I think that "**massVax**" is actually better because we are using this flag to direct people to the preregistration link at https://VaccineSignup.mass.gov

If there are other "invitation only" locations that open in the future but use a different system, then we are going to need a new flag anyway.

Reply:
sounds good!